src/content/ui-api/kubectl-gs/gitops/add-wc.md

```diff
@@ -85,14 +85,18 @@ In the most basic form the GitOps repository may be populated with Workload Clus
 without the cluster-related CRs.
 
 ```nohighlight
-kubectl gs gitops add wc \
---local-path /tmp/gitops-demo \
---name demowc \
---management-cluster demomc \
---organization demoorg \
---repository-name gitops-demo \
---dry-run
+kubectl gs gitops add workload-cluster \
+  --local-path /tmp/gitops-demo \
+  --name demowc \
```

Comment:
I was thinking if we can add a __warning__ explaining the character limitation in WC name